Applications/AdminUi/apps/admin_ui/lib/home/announcements_overview/modals/create_announcement_dialog.dart

```diff
@@ -254,18 +255,18 @@ class _AnnouncementTextFormWidget extends StatefulWidget {
   State<_AnnouncementTextFormWidget> createState() => _AnnouncementTextFormWidgetState();
 
   final String? defaultLanguage;
-  final TextEditingController _titleController = TextEditingController();
-  final TextEditingController _bodyController = TextEditingController();
-  final TextEditingController _languageController = TextEditingController();
+  final TextEditingController titleController = TextEditingController();
+  final TextEditingController bodyController = TextEditingController();
+  final TextEditingController languageController = TextEditingController();
```

Comment:
simply making this public is NOT the solution here.

if you need it above inject it and also dispose it above.

also, please move the fields above.